libraries/MySensors/MyConfig.h

```diff
@@ -35,11 +35,6 @@
 // final sketch but is helpful to see what is actually is happening during development
 //#define MY_DEBUG
 
-// Enable MY_DEBUG_VERBOSE flag for verbose debug prints related to RF24 radio.
-// Requires DEBUG to be enabled.
-// This will add even more to the size of the final sketch!
-//#define MY_DEBUG_VERBOSE
-
```

Comment:
Seems to be called MY_DEBUG_VERBOSE_RF24 now. A more specific define is fine by me.  We should perhaps keep an updated comment in MyConfig so it's easier to find.